I/ImageMagick/ImageMagick@6/build_tarballs.jl

```diff
@@ -61,7 +61,7 @@ dependencies = [
     Dependency("Zlib_jll"),
     Dependency("libpng_jll"),
     Dependency("JpegTurbo_jll"),
-    Dependency("Libtiff_jll"; compat="4.5.1"),
+    Dependency("Libtiff_jll"; compat="4.3.0"),
```

Comment:
#3024
```suggestion
    Dependency("Libtiff_jll"; compat="~4.5.1"),
```

Reply:
This is for the old (version 6) ImageMagick. Should I do the same for the new version (7)?

Reply:
Done.